config/config.go

```diff
@@ -29,6 +31,10 @@ const (
 	// DefaultFallbackConfigurationFilePath is the default fallback path that will be used to search for the
 	// configuration file if DefaultConfigurationFilePath didn't work
 	DefaultFallbackConfigurationFilePath = "config/config.yml"
+
+	// DefaultDistributedConfigurationDirectoryPath is the default path to search for additional configuration files
+	// if a custom directory has not been configured using the GATUS_CONFIG_DIR environment variable
+	DefaultDistributedConfigurationDirectoryPath = "config/config.d"
```

Comment:
Does it really make sense to have a default value for this?

Personally, I feel that the best approach would be to have either `GATUS_CONFIG_FILE` or `GATUS_CONFIG_DIR`, not both.

More specifically, when `GATUS_CONFIG_DIR` is set, we'd read every file in the folder defined by `GATUS_CONFIG_DIR`, and if not, we'd fall back to `GATUS_CONFIG_FILE`.

Also, I just want to point out that for never having written Go before, you're doing an excellent job!

Reply:
I had the default setup for many applications (f.e. nginx, sshd) in mind that use a structure like this
```
.
├── config.d
│   └── my.config
└── main.config
```
and implement distribution with `include config.d/*`-statements which are not supported by YAML. To use only `GATUS_CONFIG_DIR` with fallback to `GATUS_CONFIG_FILE`, use both `GATUS_CONFIG_DIR` and `GATUS_CONFIG_FILE` or use both but without defining a default for `GATUS_CONFIG_DIR` all seem valid to me.
I will adjust this PR during the next few days.

Reply:
Now it's reduced to single env `GATUS_CONFIG_PATH` that can contain either a file or a directory path. Are you fine with this approach?

Reply:
That's excellent!